main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player, info in players.items():
+        guild_info = info["guild"]
+
+        race_info = info["race"]
+
+        guild = Guild.objects.get_or_create(
```

Comment:
u could write here like this:
`_, guild = Guild....` so there is won't be need to use indexes at 32 line